includes/free-trial/class-wc-calypso-bridge-free-trial-marketing-changes.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Contains the logic for override WooCommerce > orders screen
+ *
+ * @package WC_Calypso_Bridge/Classes
+ * @since   2.0.1
+ * @version 2.0.1
+ */
+
+class WC_Calypso_Bridge_Free_Trial_Marketing_Changes {
+	/**
+	 * The single instance of the class.
+	 *
+	 * @var object
+	 */
+	protected static $instance = null;
+
+	/**
+	 * Get class instance.
+	 *
+	 * @return object Instance.
+	 */
+	final public static function get_instance() {
+		if ( is_null( self::$instance ) ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	public function __construct() {
+		if ( ! wc_calypso_bridge_is_ecommerce_trial_plan() ) {
+			return;
+		}
+
+		add_action( 'admin_menu', [ $this, 'remove_marketing_submenus' ], 99 );
+
+		// Add a custom class to the body tag
+		// to target the layout div.
+		add_filter( 'admin_body_class', function( $classes ) {
+			$classes .= ' free-trial-page-marketing';
```

Comment:
Quick comment: It seems this would leak the class into homescreen et al

<img width="1045" alt="image" src="https://user-images.githubusercontent.com/3747241/220802342-cb5082c1-5575-4a41-9ab4-3bd9b1a9f1ce.png">

Reply:
Thank you for catching that! I'll address it soon 👍

Reply:
🤦 Because Marketing and other WooCommerce pages are React based, this solution would not work. I'll find a different solution.

Reply:
Resolved in https://github.com/Automattic/wc-calypso-bridge/pull/984/commits/8234ffc464cea8348faca443eb1de184d4298216 and https://github.com/Automattic/wc-calypso-bridge/pull/984/commits/e9aca718b9fee68d21068260d12bf7c175a3af52

Reply:
Nice, looks good now!